darwin/exporter/formats/yolo_segmented.py

```diff
@@ -0,0 +1,236 @@
+from collections import namedtuple
+from enum import Enum, auto
+from logging import getLogger
+from multiprocessing.pool import CLOSE
+from pathlib import Path
+from typing import Iterable, List
+
+from darwin.datatypes import AnnotationFile, VideoAnnotation
+from darwin.exporter.formats.helpers.yolo_class_builder import (
+    ClassIndex,
+    build_class_index,
+    export_file,
+    save_class_index,
+)
+
+logger = getLogger(__name__)
+
+CLOSE_VERTICES: bool = False  # Set true if polygons need to be closed
+
+
+def export(annotation_files: Iterable[AnnotationFile], output_dir: Path) -> None:
+    """
+    Exports YoloV8 format as segments
+
+    Parameters
+    ----------
+    annotation_files : Iterable[AnnotationFile]
+        The ``AnnotationFile``\\s to be exported.
+    output_dir : Path
+        The folder where the new pascalvoc files will be.
+
+    Returns
+    -------
+    None
+    """
+    annotation_files = list(annotation_files)
+
+    class_index: ClassIndex = build_class_index(
+        # fmt: off
+        annotation_files, ["bounding_box", "polygon"]
+    )  # fmt: on
+
+    for annotation_file in annotation_files:
+        export_file(annotation_file, class_index, output_dir, _build_text)
+
+    save_class_index(class_index, output_dir)
+
+
+def normalise(value: float, height_or_width: int) -> float:
+    """
+    Normalises the value to a proportion of the image size
+
+    Parameters
+    ----------
+    value : float
+        The value to be normalised.
+    height_or_width : Union[float, int]
+        The height or width of the image.
+
+    Returns
+    -------
+    float
+        The normalised value.
+    """
+    return value / height_or_width
+
+
+class YoloSegmentedAnnotationType(Enum):
+    """
+    The YoloV8 annotation types
+    """
+
+    UNKNOWN = auto()
+    BOUNDING_BOX = auto()
+    POLYGON = auto()
+
+
+def _build_text(annotation_file: AnnotationFile, class_index: ClassIndex) -> str:
```

Comment:
Extensive logging which largely only shows at `DEBUG` level, so won't usually show to users, but they could opt to see.  Allows for good diagnostics if this turns out not to work as intended.